src/lfortran/semantics/ast_common_visitor.h

```diff
@@ -6423,7 +6561,10 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
                 visit_expr_list(x.m_args, x.n_args, args);
                 tmp = create_Shifta(x.base.base.loc, args);
                 return;
-            }
+            } else if (var_name == "compiler_options") {
+                tmp = create_CompilerOptions(x.base.base.loc);
```

Comment:
```suggestion
                ASRUtils::ASRBuidler b(al, x.base.base.loc);
                tmp = b.StringConstant(compiler_options.co_string, <correct-type>);
```

I don't think we even need to create a function.